README.md

```
@@ -1,78 +1,421 @@
+   ```
 
-# yarn
-yarn build
+4. **フロントエンドの実装**
+   - Services → Store → Components の順で実装
+   - コンポーザブルで再利用可能なロジックを分離
 
-# bun
-bun run build
+5. **コード品質チェック**
+   ```bash
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)